server/prisma/schema.prisma

```diff
@@ -32,13 +32,16 @@ model User {
 }
 
 model Service {
-  id          String     @id @unique @default(cuid())
-  name        String     @unique
-  description String?
-  isEnable    Boolean    @default(false)
-  createdAt   DateTime   @default(now())
-  Actions     Action[]
-  Reactions   Reaction[]
+  id             String     @id @unique @default(cuid())
+  name           String     @unique
+  description    String?
+  isEnable       Boolean    @default(false)
+  createdAt      DateTime   @default(now())
+  primaryColor   String     @default("#000000") @db.VarChar(7)
+  secondaryColor String     @default("#000000") @db.VarChar(7)
+  icon           String?
```

Comment:
You need to add a @db.varChar with a high value (Cf discord and google Token) otherwise your Image will be refused... (Believe me)

Reply:
Isn't it unlimited if not mentionned ?

Reply:
That's what i was thinking last Week, but unfortunately no